sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationServiceSerializer.cs

```diff
@@ -18,51 +18,33 @@ public static void WriteSetting(Utf8JsonWriter writer, ConfigurationSetting sett
             writer.WriteStartObject();
             writer.WriteString("key", setting.Key);
             writer.WriteString("label", setting.Label);
-
-            if (setting.IsReadOnly != default)
-            {
-                writer.WriteBoolean("locked", setting.IsReadOnly.Value);
-            }
-
-            if (setting.LastModified != default)
-            {
-                writer.WriteString("last_modified", setting.LastModified.Value.ToString(CultureInfo.InvariantCulture));
-            }
+            writer.WriteBooleanIfNotNull("locked", setting.IsReadOnly);
```

Comment:
Do we have the unit tests we need to validate that there are no changes to serialization?  IIRC many of the mock tests are using serialization that's hand-written in those tests.  I'm ok with these changes as long as we know we have full test coverage to prevent regressions.

Reply:
> IIRC many of the mock tests are using serialization that's hand-written in those tests

Yes, and all these tests fail if serialization is messed up :) I'd say serialization is the most test-covered part of our codebase.